scripts/bundleSize.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Size limit for all bundles used by each service (K)
+# Keep these +/- 10K
+min=662
+max=686
+
+services=( "news" "persian" "igbo" "yoruba" "pidgin" )
+failure=false
+
+for service in "${services[@]}"
+do
+   size=$(du -chsk build/public/static/js/{main,vendor,$service}-*.js | grep total | grep -Eo '[0-9]{1,4}')
```

Comment:
mega nit picking: `du` records how much space the files take on the file system, rather than the actual size of the files (as transferred over http for example). Running locally against the persian bundles, this is the difference between `688128` (according to du) and `679499` (counting the bytes in the file)

A possibly more representative version would be `cat build/public/static/js/{main,vendor,$service}-*.js | wc -c | tr -d ' '` (plus this would avoid all the greps)

Reply:
mega nit picks are the best nit picks 🚀

Reply:
Nice idea, Done :)